tests/test_db_store.py

```diff
@@ -34,6 +34,20 @@ def test_latest_soil_moisture_empty_database(self):
         moisture = store.latest_soil_moisture()
         self.assertIsNone(moisture)
 
+    def test_retrieve_soil_moisture(self):
+        mock_cursor = mock.Mock()
+        store = db_store.SoilMoistureStore(mock_cursor)
+        timestamp1 = str(datetime.datetime(2016, 7, 23, 10, 51, 9, 928000))
```

Comment:
Hmm so I'm highly confused about ```datetime``` and converting to strings and back, bc I guess ```strftime``` and ```strptime``` don't like time zones?! What's the Z above supposed to be? And how does that convert to pytz.utc when you make it into a ```datetime.datetime``` object?